src/Admin.php

```diff
@@ -162,6 +173,39 @@ public static function post_action_label($post_id) {
     Pdf::render($data, $labelFormat[0], $labelFormat[1]);
   }
 
+  /**
+   * Retrieves dimensions for given product.
+   *
+   * If standard product dimensions values of WooCommerce are not set,
+   * attempt to retrieve custom attributes 'Tiefe' (depth), 'Breite' (width),
+   * 'Höhe' (height).
+   *
+   * @param \WC_Product $product
+   *   Product to get the dimensions from.
+   * @param array $attributes
+   *   Product attributes.
+   *
+   * @return string
+   *   Product dimensions.
+   */
+  public static function getProductDimensions(\WC_Product $product, array $attributes): array {
+    $dimensions = $product->get_dimensions(FALSE);
+    if (count(array_filter($dimensions))) {
```

Comment:
The count is unnecessary as long as you don't test against a certain number, as the resulting condition only checks for a false-ish value.
```suggestion
    if (array_filter($dimensions)) {
```